setup.py

```diff
@@ -47,13 +47,13 @@
             'black<=19.10b0',
```

Comment:
Need to update this as well: https://github.com/openwisp/openwisp-utils/blob/6bdf528dea394562bff23fa0aeab7b6f8627c52a/setup.py#L44